### PR TITLE
Binders! (the trans kind)

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Adds binders, XL, and XS bras"

#### Purpose of change

Wanted more realism, and also more importantly having people felt seen. Also saw bras had no XL or XS variant so I fixed it.

#### Describe the solution

Adds binders, complete with spawns, variants, recipes, and XL/XS types.

#### Describe alternatives you've considered

Free top surgery!

#### Testing

Tested locally, works fine.

#### Additional context

Numbers are based on trans population and how much people are trans vs how much people felt confident reporting it, and for female_underwear_top, it was added for people who are GNC, or just don't like tits that much.

### CODE REVIEW

File: data/json/items/armor/undergarment.json [hunk 1]
```diff
@@ -159,6 +159,158 @@
...
+        "append": true
+      }
+    ]
+  },
+  {
+    "id": "xl_binder_top",
+    "type": "ARMOR",
+    "copy-from": "binder_top",
+    "name": { "str_sp": "XL binder" },
+    "description": "A binder, meant to bind the chest close to the body to keep a persons chest flat, but custom made to accomodate larger mutants.",
```
Comment (Knut-Aage-Hofseth) [inline data/json/items/armor/undergarment.json:300]:
```suggestion
    "description": "A binder, meant to bind the chest close to the body to keep a persons chest flat, but custom made to accommodate larger mutants.",
```

File: data/json/items/armor/undergarment.json [hunk 2]
```diff
@@ -159,6 +159,158 @@
...
+    "proportional": { "weight": 1.5, "volume": 1.5, "price": 1.25 },
+    "extend": { "flags": [ "OVERSIZE" ] }
+  },
+  {
+    "id": "xs_binder_top",
+    "type": "ARMOR",
+    "copy-from": "binder_top",
+    "looks_like": "binder_top",
+    "name": { "str_sp": "XS binder" },
+    "description": "A binder, meant to bind the chest close to the body to keep a persons chest flat, but custom made to accomodate smaller mutants.",
```
Comment (Knut-Aage-Hofseth) [inline data/json/items/armor/undergarment.json:310]:
```suggestion
    "description": "A binder, meant to bind the chest close to the body to keep a persons chest flat, but custom made to accommodate smaller mutants.",
```

File: data/json/items/armor/undergarment.json [hunk 3]
```diff
@@ -159,6 +159,158 @@
...
+        "id": "binder_top_floral",
+        "name": { "str": "floral binder" },
+        "description": "This one has a floral pattern.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "binder_top_dinos",
+        "name": { "str": "dinosaur binder" },
+        "description": "This one has a cartoon rendition of a stegosarus on it.",
```
Comment (Knut-Aage-Hofseth) [inline data/json/items/armor/undergarment.json:268]:
```suggestion
        "description": "This one has a cartoon rendition of a stegosaurus on it.",
```

**Review by Knut-Aage-Hofseth — COMMENTED** [review]
Just the spelling errors that the bot caught

**Comment by Zireael07** [comment]
I know that regular females also sometimes wear binders, so +1 to having those in game

File: data/json/items/armor/undergarment.json (hunk 2 above)
Comment (Paprika-H) [reply data/json/items/armor/undergarment.json:310]: You are my savior lmao

**Comment by Paprika-H** [comment]
> I know that regular females also sometimes wear binders, so +1 to having those in game

What does that "regular females" part mean?

**Pull request merged**